app/logic/emailHandler.py

```diff
@@ -200,29 +196,27 @@ def build_email(self):
         return (template_id, subject, body)
 
     def send_email(self):
-        defaultEmailInfo = {"senderName":"Sandesh", "replyTo":self.reply_to}
+        defaultEmailInfo = {"senderName":"CELTS", "replyTo":"celts@berea.edu", "senderAddress":"celts@berea.edu"} 
```

Comment:
We have a `celts_admin_contact` in our config that is set to "celts@berea.edu". Should we use that config item everywhere? or rename it and use it?